Workshop 2025: Schedule talks

MakisH opened the pull request:
I scheduled the talks taking into account:
- Individual restrictions: So far, only Ingo needs to present on Wed evening or Thu morning
- Prioritizing user talks in the first day, which is already dev-heavy
- Spreading the more high-level talks across sessions
- Scheduling the developers in each session last

Pending: chairs and coordinators for each session.

MakisH committed:
Workshop 2025: Schedule talks

MakisH committed:
Fix time discontinuity

MakisH commented:
@uekerman noticed a time imbalance in the slots, and we already needed a break in Wed afternoon. I moved Anthony's talk to Thursday and introduced another coffee break. Rendered:

![Screenshot 2025-07-07 at 17-20-59 preCICE workshop 2025 preCICE - The Coupling Library](https://github.com/user-attachments/assets/6df20512-7ff3-4b50-943e-8a1a6f8dfab3)

MakisH committed:
Merge branch 'master' into workshop2025-schedule

uekerman commented on pages/community/precice-workshop-2025.md:
```diff
@@ -86,148 +86,155 @@ Preliminary schedule:
+    </details>
+  - 16:30-17:00 Coffee break
+  - 17:00-18:00 Talks
+    *Chair: Gerasimos Chourdakis*
```
```suggestion
    <br/>*Chair: Gerasimos Chourdakis*
```

uekerman reviewed (approved):
Looks good to me. I mainly looked at the screenshot.

Concerning the course: I guess we will be agile with the timing, right? Starting with module 3 on Monday could make sense if possible and maybe starting with module 5 a little bit later on Tue (14:00?) if necessary.

Should we also add coffee breaks on Monday and Tuesday to avoid misunderstandings with the caterer?

MakisH committed:
Update pages/community/precice-workshop-2025.md

Co-authored-by: Benjamin Uekermann <benjamin.uekermann@gmail.com>

MakisH committed:
Add coffee and informal dinner slots, course details

MakisH commented:
Current course program in detail:

- Monday (course):
  - 12:00-13:00: 🍜 *Registration and lunch*
  - 13:00-13:30: Setting up for the [Training course](community-training.html)
  - 13:30-15:30: Course part 1 (Basics) - Benjamin Uekermann
  - 15:30-16:00: ☕️ *Coffee break*
  - 16:00-18:00: Course part 2 (Tools) - Gerasimos Chourdakis
  - 19:00-22:00: 🍻 *Time for community-organized social events*
- Tuesday (course):
  - 08:30-09:00: ☕️ *Arrival and coffee*
  - 09:00-11:00: Course part 3 (Implicit coupling) - Gerasimos Chourdakis
  - 11:00-12:00: Course part 4a (Data mapping) - Benjamin Uekermann
  - 12:00-13:00: 🍜 *Lunch*
  - 13:00-14:30: Course part 4b (Data mapping) - Benjamin Uekermann
  - 14:30-15:00: ☕️ *Coffee break*
  - 15:00-17:00: Course part 5 (FSI workflow) - Claudio Caccia (community training block)
  - 17:00-18:00: Course recap and Q&A
  - 19:00-22:00: 🍻 *Time for community-organized social events*

I updated the schedule to have this state. Of course, if we are extremely early or late, we can adapt on the fly.

From experience, the beginning is slower, and we always take some time resolving individual issues. That's why I think we should not overload the first day.

The main issue I see here is that the more complicated parts are all together on Tuesday.

___

If you think we can go faster, we could offer a setup session before lunch on Tuesday and split part 3 between the two days. We can use the left-over time for a first support session or extended Q&A. But I am not enthusiastic about this option.

Monday:

- 11:00-12:00: Setting up (optional) -> We are going to be there anyway
- 12:00-13:00: Lunch
- 13:00-15:00: Part 1 (2h)
- 15:00-15:30: Coffee break
- 15:30-17:00: Part 2 (1.5h)
- 17:00-18:00: Part 3a (1h)

Tuesday:

- 09:00-10:00: Part 3b (1h)
- 10:00-12:00: Part 4 (2h)
- 12:00-13:00: Lunch
- 13:00-15:00: Part 5 (2h)
- 15:00-15:30: Coffee break
- 15:30-16:00: Recap and Q&A
- 16:00-18:00: Support session

MakisH committed:
Merge branch 'master' into workshop2025-schedule
